src/ErrorLogger/Error.js

```diff
@@ -9,60 +9,47 @@ import * as ACT from './actions';
 import * as SC from './styles';
 
 const ErrorComponent = (props) => {
-  const { error, errorProps, errorNav } = props;
+  const { error, errorNav } = props;
   const [specErrorVisible, setspecErrorVisible] = useState(false);
 
   const handleClickSpecError = () => {
     setspecErrorVisible(!specErrorVisible);
   };
 
-  const componentProps = {
-    borderBottom: errorProps.ERROR_BORDER_BOTTOM,
-  };
-
-  const fileProps = {
-    errorFile: errorProps.ERROR_FILE,
-    errorFileHover: errorProps.ERROR_FILE_HOVER,
-  };
-
   const typeProps = {
     onClick: handleClickSpecError,
-    errorType: errorProps.ERROR_TYPE,
   };
 
   const shortMessageProps = {
     onClick: handleClickSpecError,
-    shortMessage: errorProps.ERROR_SHORT_MESSAGE,
   };
 
   const fullMessageProps = {
-    fullMessage: errorProps.ERROR_FULL_MESSAGE,
   };
 
   const errorArrowProps = {
     expanded: specErrorVisible,
     onClick: handleClickSpecError,
-    errorArrow: errorProps.ERROR_EXPAND_ARROW,
   };
 
   return (
-    <SC.ErrorComponent {...componentProps}>
+    <SC.ErrorComponent className={'errorComponent'}>
```

Comment:
Do all these need to be `className="..."` instead of `className={'...'}`?

Reply:
Yes, they need to be  `className="..."`. I'll change them!